docs/docs/usage.md

```diff
@@ -177,4 +177,7 @@ Triggers the build process of the specified module according to the implicit or
 | `-s (--source)`   | Optional  | The path to the folder where the project’s `mta.yaml` file is located; the current path is set as default.                              |`mbt module-build -m=my_module  -s=C:/TestProject`
 | `-t (--target)`   | Optional  | The folder where the module build results will be saved. If this parameter is not provided, the module build results are saved in the `<current folder>/.<projectname>_mta_build_tmp/<module name>` folder. If  the parameter is provided, the build results are saved in the root of the folder provided by the argument. <br><br><b>Note:</b><br>If the `--target` parameter is used when building several modules, make sure that the names of the specified modules' build artifacts are unique. You can configure the build artifact name using the `build-artifact-name` build parameter as described [here](configuration.md#configuring-the-build-artifact-name). | `mbt module-build -m=my_module  -t=C:/TestProject/build_results_tmp`<br> <br>The module’s build results will be saved directly in the `C:/TestProject/build_results_tmp/` folder
-| `-e (--extensions)`   | Optional  | The path or paths to multitarget application extension files (`.mtaext`). Several extension files separated by commas can be passed with a single flag, or each extension file can be specified with its own flag.| `mbt module-build -m=my_module -e=test1.mtaext,test2.mtaext`<br>or<br>`mbt module-build -m=my_module -e=test1.mtaext -e=test2.mtaext`
\ No newline at end of file
+| `-e (--extensions)`   | Optional  | The path or paths to multitarget application extension files (`.mtaext`). Several extension files separated by commas can be passed with a single flag, or each extension file can be specified with its own flag.| `mbt module-build -m=my_module -e=test1.mtaext,test2.mtaext`<br>or<br>`mbt module-build -m=my_module -e=test1.mtaext -e=test2.mtaext`
+| `-g (--mtad-gen)`   | Optional  | If the parameter is provided, the deployment descriptor `mtad.yaml` is generated by default in the current folder or in the folder configured by the `--target` parameter. <br> A module's `path` property in the generated `mtad.yaml` file points to the module's build results if this module was selected using the `--modules` option. <br><br> <b>Notes</b>:<ul><li>The selected module list specified using the `--module` option, does not affect the list of modules in the resulting `mtad.yaml` file. The `mtad.yaml` file is always generated according to the default Cloud MTA Builder settings, the `build-parameters` configurations in the `mta.yaml` file (e.g. `supported-platforms`), and the selected target platform.<li>By default, the `mtad.yaml` is generated for the `cf` target platform. You can configure a different target plaform using the `--platform` option.  | `mbt module-build -m=my_module1, my_module2 –g`
```

Comment:
mbt module-build -m=my_module1, my_module2 –g
please care to remove space between modules or put them with own -m flag or with ""

Reply:
fixed